### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
### Below are listed all the features included in this PR:

- [ ] Fix header and navbar semantics.
- [ ] Add a responsive desktop version.

### CODE REVIEW

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @cuauh-cabrera, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your work on this project is outstanding. 🥳🥳🥳 ✔


## Required Changes ♻️

- [x] So far, you've done an admirable job. However, there are a few things I'd like to point out to ensure that your project looks as close to the template as possible. :ok_hand: 😊
### Your cards :point_down: 
Screenshot from 2023-06-08 16-04-38

### Cards on the template :point_down: 
Screenshot from 2023-06-08 16-05-01



1. The color of the titles and languages on the cards differs from the color used in the template. Please correct that so that the colors match. Instead, use `#172B4D`.
2. Please fix the layout of the languages in the card as well, because the list of languages in the template has some margins to the left and right. 
In this case, what I would recommend to make the position of the languages list similar to the template is: :point_down: 
- to remove the `width: 100%;` you gaved to the `.projects__card--tags` class. 
- (**optional**) I think you can remove the `display: flex; ` and the `flex-direction: row; ` form the  `.projects__card--tags` class because I think you don't need it there.
- then add a `gap: 10px` to `.projects__card--tags ul`.

3. Please ensure that your form resembles the template by including a lastname input in the same row as the firstname input. 
### Your form :point_down: 
Screenshot from 2023-06-08 16-37-40

### The template :point_down: 
Screenshot from 2023-06-08 16-38-09

4. Last but not least, Kindly assure that the space between the links in the navigation bar and their colour match the template.  I think this `#172B4D` is the colour you should use.
### Your navigation :point_down: 
Screenshot from 2023-06-08 16-52-51

### The template :point_down: 
Screenshot from 2023-06-08 16-51-38


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -13,47 +13,52 @@
...
-                                alt="This is a Twitter icon. Click on it to visit my Twitter profile"
-                                class="headline__icons--twitter"></a></li>
-                    <li><a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
-                                alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
-                                class="headline__icons--linkedin"></a></li>
-                    <li><a href="https://medium.com"><img src="./assets/medium.png"
-                                alt="This is a medium icon. Click on it to visit my medium profile"
-                                class="headline__icons--medium"></a></li>
-                    <li><a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
-                                alt="This is a github icon. Click on it to visit my github profile"></a></li>
```
Comment (bobb-Rob) [inline index.html:43]: - [ ] Your project has some issues with the work/project sections as mentioned by the previous reviewer, the image below best describes what the issues are 👇🏾 

The issue is from many angles, please try to fix it

Mobile
image

Desktop
image


Please fix it.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @cuauh-cabrera, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (cuauh-cabrera) [reply index.html:43]: @bobb-Rob Thanks for your feedback. As per the images attached, I can’t see the issue with the project's card. In the images below is not possible to see the problem in the mobile version, please note that the images come from three different web browsers (Firefox, Chrome, and Safari). 
Captura de Pantalla 2023-06-09 a la(s) 12 49 31
Captura de Pantalla 2023-06-09 a la(s) 12 54 44
Captura de Pantalla 2023-06-09 a la(s) 12 56 07

**Comment by cuauh-cabrera** [comment]
@bobb-Rob same situation with the about me section cards, in the images above is not possible to see the issue you mention.
Captura de Pantalla 2023-06-09 a la(s) 12 57 40

Captura de Pantalla 2023-06-09 a la(s) 12 59 11

Captura de Pantalla 2023-06-09 a la(s) 12 59 42

* **Same situation with the About me section in the Desktop version:**

Captura de Pantalla 2023-06-09 a la(s) 13 03 56

Captura de Pantalla 2023-06-09 a la(s) 13 05 09

Captura de Pantalla 2023-06-09 a la(s) 13 05 43

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @cuauh-cabrera , 👋 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors. Please fix them and resubmit for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by roshan-bajgain — APPROVED** [review]
Hi @cuauh-cabrera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**